docs/application/native/guides/internationalization/i18n.md

```diff
@@ -1992,6 +1995,9 @@ To generate and manage an alphabetic index:
      ```
      i18n_alpha_idx_add_labels(alpha_idx, "ko", "KR");
      ```
+     > **Note**  
+     > This function is deprecated from Tizen 5.0. Instead, use `i18n_alpha_idx_add_labels_with_locale_id()`
```

Comment:
Please add comma after `i18n_alpha_idx_add_labels_with_locale_id()` to get the consistency.